api/v1alpha1/virtualmachine_conversion_test.go

```diff
@@ -321,6 +322,196 @@ func TestVirtualMachineConversion(t *testing.T) {
 		g.Expect(spoke.Status.Phase).To(Equal(v1alpha1.Created))
 	})
 
+	t.Run("VirtualMachine hub-spoke Status Conditions", func(t *testing.T) {
+		hub := nextver.VirtualMachine{}
+		loc, err := time.LoadLocation("UTC")
+		g.Expect(err).ToNot(HaveOccurred())
+		now := metav1.Date(2000, time.January, 1, 0, 0, 0, 0, loc)
+
+		vmClassCond := metav1.Condition{
+			Type:               nextver.VirtualMachineConditionClassReady,
+			Status:             metav1.ConditionTrue,
+			LastTransitionTime: metav1.NewTime(now.AddDate(1, 0, 0)),
+		}
+		vmImageCond := metav1.Condition{
+			Type:               nextver.VirtualMachineConditionImageReady,
+			Status:             metav1.ConditionTrue,
+			LastTransitionTime: metav1.NewTime(now.AddDate(2, 0, 0)),
+		}
+		vmSetRPCond := metav1.Condition{
+			Type:               nextver.VirtualMachineConditionVMSetResourcePolicyReady,
+			Status:             metav1.ConditionTrue,
+			LastTransitionTime: metav1.NewTime(now.AddDate(3, 0, 0)),
+		}
+		vmBSCond := metav1.Condition{
```

Comment:
`vmBSCond` at a first glance seems to be a **BS** condition like `Foo`, but un fact is **BootstrapReady**. 🤣

Reply:
It goes along nicely with the `bsArgs` that's used elsewhere. Just a reflection of how I was feeling at the time :)